doc/userguide/src/ExecutingTestCases/ConfiguringExecution.rst

```diff
@@ -130,6 +130,14 @@ combining individual tags or patterns together::
    --exclude xxORyyORzz
    --include fooNOTbar
 
+It is also possible to use the reserved tag `robot:exclude` to achieve
+the same effect as with using the `--exclude` option::
+
+    *** TEST CASES ***
+    Test Case
+        [Tags]    robot:exclude
+        Fail      this test will be excluded
+
```

Comment:
1. Should mention this functionality is new in RF 5.0. Could add an explicit note about that, but perhaps here the sections could just start like "Staring from Robot Framework 5.0, it is also possible ...".
2. Robot code should be formatted. You can use `.. sourcecode:: robotframework` directive to enable that.
3. We have used `*** Test Cases ***` style elsewhere and for consistency it would be good here as well.
4. Perhaps the test could be named "Example".